app/services/table_of_contents_builder.rb

```diff
@@ -58,6 +58,7 @@ def expand?(component)
     if component["id"] == document["id"]
       true
     elsif @online_content && component["has_online_content_ssim"]&.first == "true" && !@found_online_content
+      # component["has_online_content_ssim"]&.first turns false before it hits the online component - shaune
```

Comment:
I don't really understand what this means.

Reply:
That was a note for @eliotjordan  as he was reviewing since I noticed strange behavior that was probably related to the bug in the JS code.  Meant to delete it... it's been removed.